yente/settings.py

```diff
@@ -153,6 +154,15 @@ def random_cron() -> str:
 # Matcher defaults:
 DEFAULT_ALGORITHM = env_str("YENTE_DEFAULT_ALGORITHM", "logic-v1")
 BEST_ALGORITHM = env_str("YENTE_BEST_ALGORITHM", "logic-v1")
+HIDDEN_ALGORITHMS = [
+    algo for algo in env_str("YENTE_HIDDEN_ALGORITHMS", "").split(",") if algo != ""
+]
+# Ensure that all hidden algorithms are valid
+assert all(
+    algo in [a.NAME for a in nomenklatura.matching.ALGORITHMS]
+    for algo in HIDDEN_ALGORITHMS
+), "Invalid algorithm name in YENTE_HIDDEN_ALGORITHMS"
```

Comment:
I think this assertion is fair - better to find out that the algo we mean to hide isn't hidden - if we try to deploy an instance that doesn't start up, I think it won't scale down the old nodes. That's built into `dokku` as well, which gives me the idea that it can be expected from people's deployment processes.